rules/prefer-object-define-properties.js

```diff
@@ -0,0 +1,179 @@
+'use strict';
+const isSameReference = require('./utils/is-same-reference.js');
+const {removeArgument} = require('./fix/index.js');
+
+const MESSAGE_ID = 'prefer-object-define-properties';
+const messages = {
+	[MESSAGE_ID]: 'Prefer `{{replacement}}` over multiple `{{value}}`.',
```

Comment:
This message may cause confusing, since the "secondCallExpression" can be `Object.defineProperties`, we need improve it.